packages/mui-material/src/ButtonBase/ButtonBase.js

```diff
@@ -68,6 +68,21 @@ export const ButtonBaseRoot = styled('button', {
   },
 });
 
+function useRippleHandler(ripple, rippleAction, eventCallback, skipRippleAction = false) {
+  return useEventCallback((event) => {
+    if (eventCallback) {
+      eventCallback(event);
+    }
+
+    const ignore = skipRippleAction;
+    if (!ignore) {
```

Comment:
While you're touching this, you could also remove that variable, the first name is already semantically correct.